UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs

```diff
@@ -40,5 +40,9 @@ public class VolumePerLengthTests : VolumePerLengthTestsBase
         protected override double LitersPerKilometerInOneCubicMeterPerMeter => 1E6;
 
         protected override double LitersPerMillimeterInOneCubicMeterPerMeter => 1;
+
+        protected override double UsGallonsPerMileInOneCubicMeterPerMeter => 4.25143707430272e5;
```

Comment:
Verified
https://www.wolframalpha.com/input?i=1.0000000000000000+cubic+meter+per+meter+to+us+gallon+per+mile